packages/cli/src/tools/PackageManager.js

```diff
@@ -21,11 +21,7 @@ export default class PackageManager {
   }
 
   shouldCallYarn() {
-    return (
-      !this.options.forceNpm &&
-      yarn.getYarnVersionIfAvailable() &&
-      yarn.isGlobalCliUsingYarn(this.options.projectDir)
```

Comment:
why this was removed? `projectDir` is used by other commands and, while with `init` we have not reliable way to say which package manager to use by default, other commands have this data (`yarn.lock` exists or not).

Reply:
That's true. Will change that.

Reply:
Depending on https://github.com/react-native-community/react-native-cli/pull/243